SIMDExprTree.py

```diff
@@ -0,0 +1,134 @@
+__author__ = 'Ken Sible'
```

Comment:
Please add a header consistent with other NRPy+ Python modules that contains author info and briefly describes the purpose of the module.